core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/rules/SnowflakeTimeUnits.scala

```diff
@@ -0,0 +1,39 @@
+package com.databricks.labs.remorph.parsers.snowflake.rules
+
+object SnowflakeTimeUnits {
```

Comment:
is it optimizer subclass?

Reply:
I haven't considered this idea, as the intent of `SnowflakeTimeUnits` was initially merely to hold the mapping between Snowflake's numerous aliases for date/time parts to their DB SQL counterpart. I could make it a full-fledged optimizer, specialized on `CallFunction` instances that have a date/time part argument though I reckon maintenance would be easier if we keep all the optimizations-that-care-about-CallFunction in a single place. Again, your call...

Reply:
okay, can stay as object. just think how call mapper could be broken down to smaller classes. `optimizer.Rules` is one of the scalable approaches.